lib/cog_api/http/groups.ex

```diff
@@ -3,17 +3,63 @@ defmodule CogApi.HTTP.Groups do
 
   alias CogApi.Endpoint
   alias CogApi.Resources.Group
+  alias CogApi.Resources.User
 
   def index(%Endpoint{}=endpoint) do
     Base.get(endpoint, "groups") |> Base.format_response("groups", [%Group{}])
   end
 
   def show(%Endpoint{}=endpoint, id) do
-    Base.get(endpoint, "groups/#{id}") |> Base.format_response("group", %Group{})
+    group_json = [
+      Task.async(fn -> get_group(endpoint, id) end),
+      Task.async(fn -> get_users_for_group(endpoint, id) end),
+    ]
+
+    {group_status, group} = group_json |> List.first |> Task.await
+    {users_status, users} = group_json |> List.last |> Task.await
+
+    {
+      Base.response_type([group_status, users_status]),
+      %{group | users: users}
+    }
+  end
+
+  defp get_users_for_group(endpoint, id) do
+    response = Base.get(endpoint, "groups/#{id}/memberships")
+    json_map = %{"members" => %{"users" => [%User{}]}}
+    users = Poison.decode!(response.body, as: json_map)["members"]["users"]
+    {
+      Base.response_type(response),
+      users
+    }
+  end
+
+  defp get_group(endpoint, id) do
```

Comment:
Can this be `Base.get(endpoint, "groups/#{id}") |> Base.format_response("group", %Group{})`?